### PR TITLE
Added "on-shown" attribute

### PR DESCRIPTION
**Pull request by syabro**

### CODE REVIEW

File: src/uiSelectController.js [hunk 1]
```diff
@@ -112,6 +112,10 @@ uis.controller('uiSelectCtrl',
 
       $scope.$broadcast('uis:activate');
 
+      $timeout(function(){
```
Comment (wesleycho) [inline src/uiSelectController.js]: Is it necessary to do a $timeout?

**Comment by wesleycho** [comment]
This needs changes to `dist` removed.

File: src/uiSelectController.js (hunk 1 above)
Comment (syabro) [inline src/uiSelectController.js]: Honestly I don't remember, but I guess it was added to be sure it's triggered after the main cycle.

**Comment by syabro** [comment]
@wesleycho what do you mean by 'dist'? :)

**Comment by wesleycho** [comment]
Click on the `Files changed` tab of this PR and you'll see what I mean.

**Comment by syabro** [comment]
Done, @wesleycho 
Sorry for a delay

**Comment by user378230** [comment]
Instead of adding separate timeout, couldn't you just call lower down...?

```js
var container = $element.querySelectorAll('.ui-select-choices-content');
      if (ctrl.$animate && ctrl.$animate.on && ctrl.$animate.enabled(container[0])) {
        ctrl.$animate.on('enter', container[0], function (elem, phase) {
          if (phase === 'close') {
            // Only focus input after the animation has finished
            $timeout(function () {
              // <<<<<<<<<<<<<< HERE <<<<<<<<<<<<<<
              ctrl.focusSearchInput(initSearchValue);
            });
          }
        });
      } else {
        $timeout(function () {
          // <<<<<<<<<<<<<< AND HERE <<<<<<<<<<<<<<<
          ctrl.focusSearchInput(initSearchValue);
          if(!ctrl.tagging.isActivated && ctrl.items.length > 1) {
            _ensureHighlightVisible();
          }
        });
      }
```

**Comment by user378230** [comment]
Superseded by #1723

**Pull request closed without merging**